mobile/app/build.gradle

```diff
@@ -24,15 +24,14 @@ def metaFDroid(outputFileName) {
 }
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion = '29.0.3'
+    compileSdkVersion 33
 
     defaultConfig {
         applicationId "org.exarhteam.iitc_mobile"
-        minSdkVersion 17
-        targetSdkVersion 29
+        minSdkVersion 23
+        targetSdkVersion 33
         versionCode = getVersionCodeTimeStamps()
-        versionName "0.34.1"
+        versionName "0.40.1"
```

Comment:
Please remove the version change, since a new number will be assigned when a new version of the IITC is released

Reply:
Ok.. i understand.. will do it